solvers/qpSpline/ContinuityConstraint.cpp

```diff
@@ -0,0 +1,26 @@
+#include "ContinuityConstraint.h"
+#include <cassert>
+
+ContinuityConstraint::ContinuityConstraint(int derivative_order, int first_spline_index, int second_spline_index) :
```

Comment:
i would be much nicer to have this interface be compatible with our general constraint interface.  In other words, this should (eventually) derive from LinearConstraint, and you should move the constraint evaluation that you have inline in the code below into this class so it can return A and b.

If that's too much work, i would be ok accepting as in and putting in a new ticket for that.